### ISSUE TITLE
Lured Pokestops not showing

### ISSUE THREAD

**Issue by rulzAT** [RocketMap/RocketMap#398]
Today i recognized that lured Pokestops aren't showing anymore. The night before all worked fine. It seems that with the new in game "location" feature the information about lured pokemon and stops aren't readable anymore. 
See here:
![image](https://cloud.githubusercontent.com/assets/20883342/17545529/d6975216-5edd-11e6-98bb-6c4874d9f87d.png)

i update to the latest dev version. now i see only the stops outside the scanning area. sometime when i zoom in i see one and it dispers within a second.

![image](https://cloud.githubusercontent.com/assets/20883342/17546310/cb7041c8-5ee1-11e6-8967-d5bbdc3484b9.png)

**Comment by Itnain** [comment]
same here. Can't see lured poketstop near me. But I see some far away lured poketstops that are expired in over 2 days (48 hours!).
![lured_bug](https://cloud.githubusercontent.com/assets/20833730/17546727/26540c78-5eec-11e6-828d-b8738e99b25b.png)

**Comment by mattskey** [comment]
I'm seeing this as well.

It was working yesterday on the 2.2 release.
I started to see it today, so I moved to the latest dev release, but it's seen as well.

The lured PS appear then disappear after a few seconds.
also, if a Pokemon appears at the lure it notifies me, even though the lure isn't there, I click on the notification and I am taken to the co-ordinates on the map, but no Pokemon or lured PS can be seen.

Something changed on Niantic end?

**Comment by mi49** [comment]
^^^^^^ 
also have that problem,lured pokestop apper then disappear after 5 seconds

**Comment by mattskey** [comment]
I gave  "npm cache clear" a go, no luck.

**Comment by johnkernke** [comment]
I think this is something API related, I have been noticing that Lured Pokestops only appear if they were in the last scanned area. Once the scanner moves on to the next area it will overwrite the stop as not lured.

**Comment by rulzAT** [comment]
i think this is caused by the new tracking feature. Since the update i see only "normal" spawned pokemon in the ingame near by. 
I think lured pokemon aren't sent by the server anymore.

**Comment by BaIthamel** [comment]
+1 this issue :(

**Comment by SkOODaT** [comment]
+1 this issue too :( any progress on a fix sorta a major feature considering all the good pokes show up at lures lol

**Comment by BlazeMuisNL** [comment]
Same issue right here..

**Comment by delenca** [comment]
I seem to be able to see lured pokemon just no stops.

**Comment by SkOODaT** [comment]
If this is an api change would it be better to post in the Api git issues tracker ? Or is this something with the map ? When I move the marker I see the lured stop pop but only for a second and then it disappears .... Not sure about still seeing lured Pokemon will test that later when I'm out ~

**Comment by LordRad** [comment]
as @delenca states lured pokemon are showing, its just the pokestop that its presenting the issue; currently unable to post screenies (office proxy issue)

**Comment by blastagator** [comment]
I disagree, I'm at d0994b475586347bab76a11821534763546a3ebc and the pokestop shows as unlured w/ no pokemon.  The three on the right are lured.
![d0994b4](https://cloud.githubusercontent.com/assets/7576727/17572827/9adc5e50-5f24-11e6-92fc-efe0b68d455d.jpg)

I am updating now to latest commit, but it doesn't look like any of the recent commits would have an effect on this behavior, but I will be sure to update.  (Maybe something in the pogoapi update.)

**Comment by PorkScripts** [comment]
Having the same issue. I am finding a few lured pokestops & pokemon but a lot less than I should be. This started happening last night.

Also, @LordRad and @delenca I don't think that's even possible. To get lured pokemon with the current API, you have to look within the pokestop's lure info. Meaning you can't get lured pokemon and not lured pokestops because they are packaged together.

**Comment by machadox** [comment]
pokemons from lure still not show? :(

**Comment by corycomer** [comment]
Yeah, I can see the lured pokestops again, but not able to see lured pokemon. I compared against another app to be sure there were actually pokemon, so it looks like they're still in the response, but the format might have changed or something.

**Comment by blastagator** [comment]
If you linked the source code for the working app, I suspect that would be very helpful for them to fix the issue in this app.

**Comment by corycomer** [comment]
Sorry, should have mentioned it. I checked Go Radar, the iPhone app (https://itunes.apple.com/us/app/go-radar-live-map-for-pokemon/id1139995553?mt=8) and I noticed they have a web version (https://map.goradar.io/#) from the looks of it, it might be an older or modified version of PokemonGo-Map, so probably not very helpful :/ 

I was using it last night in an area with a lot of lures side-by-side with PokemonGo-Map running on my local machine at home and when comparing the maps, the Go Radar one had lured Pokemon while my local copy didn't. Made sure I was on the latest commit when I got home, rebuilt the static assets, made sure all the pip deps were up to date but still didn't see any lured pokemon.
